### ISSUE TITLE
[Bug] Immich statistics URL changed in v1.85.0

### ISSUE THREAD

**Issue by gabe565** [gethomepage/homepage#2282]
### Description

Upgrading Immich to v1.85.0 causes the Homepage widget to show an API error.

![image](https://github.com/gethomepage/homepage/assets/7717888/497bcfbd-67e8-4747-a1fe-615cb8481497)

This is due to immich-app/immich#4817 which changed the statistics URL from `/api/server-info/stats` to `/api/server-info/statistics`.

### Steps to reproduce

1. Upgrade Immich to v1.85.0
2. Configure Immich widget

### homepage version

v0.7.4

### Installation method

Docker

### Troubleshooting

n/a

### Before submitting, I have made sure to

- [X] Check [the documentation](https://gethomepage.dev/)
- [X] Follow [the troubleshooting guide](https://gethomepage.dev/latest/more/troubleshooting/) (please include output above if applicable).
- [X] Search [existing issues](https://github.com/gethomepage/homepage/search?q=&type=issues) and [discussions](https://github.com/gethomepage/homepage/search?q=&type=discussions).

**Comment by shamoon** [comment]
Why

**Comment by gabe565** [comment]
Yeah it's not great. I honestly considered opening an issue in the Immich repo. Should I try going that route first? It's possible that it was a typo that got lost in in the PR changes.

**Comment by shamoon** [comment]
I mean sure? I dont get why they would merge a complete breaking change for that...

**Comment by gabe565** [comment]
Agreed. I'll open and issue and report back.

**Comment by shamoon** [comment]
Thanks for opening the issue there. I'll think about another solution in the meantime

**Comment by shamoon** [comment]
#2284 will sort this. Immich api supports checking the version so we'll just use that for now to choose the right endpoint. Can re-address if they change things on their end

**Comment by spupuz** [comment]
so is it solved? how do i solve in my installation?

**Comment by Chacsam** [comment]
> so is it solved? how do i solve in my installation?

updating to v8 solved it for me today
